src/helpers.rs

```diff
@@ -194,7 +194,7 @@ pub fn get_install_path<P: AsRef<Path>>(install_path: Option<P>) -> Option<PathB
     // Local executable dir if no cargo is found
     if let Some(d) = dirs::executable_dir() {
```

Comment:
```suggestion
    let dir = dirs::executable_dir();
    
    if let Some(d) = &dir {
        debug!("Fallback to {}", d.display());
    }

    dir
```

IMO it would be better to avoid `return` in `if`.